app/javascript/packs/red_hat_cloud_services.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect, useReducer }  from 'react';
+import { TablePfProvider, Table, Filter, FormControl, Button, Checkbox, Paginator, PAGINATION_VIEW } from 'patternfly-react'
+import orderBy from 'lodash/orderBy'
+import RedHatCloudServicesTable from './red_hat_cloud_services_table';
+import { reducer, selectRow } from './reducers';
+import {  setPageWrapper, paginate } from './helper'
+
+const filterFields = [
+  {
+    id: 'name',
+    title: 'Name',
+    placeholder: 'Filter by Name',
+    filterType: 'text'
+  },
+  {
+    id: 'type',
+    title: 'Type',
+    placeholder: 'Filter by Provider Type',
+    filterType: 'text'
+  }
+];
+
+const columns = [
+  {
+    label: 'Name',
+    property: 'name'
+  },
+  {
+    label: 'Type',
+    property: 'type'
+  },
+  {
+    label: 'Action',
+    property: 'action'
+  },
+];
+
+
+const RedHatCloudServices = (_props) => {
```

Comment:
if you don't intend to use props just don't access the argument `const Foo = () => {...}`